test/sandbox-test.js

```diff
@@ -243,6 +243,28 @@ describe("sinonSandbox", function () {
         }
     });
 
+    describe(".restore", function () {
+        it("throws when passed arguments", function () {
+            this.sandbox = sinonSandbox.create();
+
+            var failed = false;
```

Comment:
The only keys on assert seem to be:

```
[ 'toString',
  'same',
  'equals',
  'greater',
  'less',
  'defined',
  'isNull',
  'match',
  'isObject',
  'isFunction',
  'isTrue',
  'isFalse',
  'isString',
  'isBoolean',
  'isNumber',
  'isNaN',
  'isArray',
  'isArrayLike',
  'keys',
  'exception',
  'near',
  'hasPrototype',
  'contains',
  'tagName',
  'className',
  'spy',
  'stub',
  'mock',
  'fakeServer',
  'clock',
  'fakeServerWithClock' ]
```

`assert.exception` was exactly what I was looking for.